index.html

```diff
@@ -6802,6 +6802,31 @@ <h3>Fitness for Purpose</h3>
 [=verifiers=] unless they accept the liability of ignoring the policy.
         </p>
       </section>
+
+      <section class="informative">
+        <h3>Artificial Intelligence</h3>
+
+        <p>
+Systems using artificial intelligence might be capable of performing
+complex tasks at a level that meets or exceeds human performance.
+This might include tasks such as the acquisition and use of
+[=verifiable credentials=]. Therefore, using such tasks as a signal of
+human activity might not be accurate.
```

Comment:
```suggestion
[=verifiable credentials=]. Therefore, performing such tasks might be unreliable
as a signal of human activity or "CAPTCHA".
```

(Editorial bikeshedding, feel free to ignore)

Reply:
Would you be opposed to noting that "CAPTCHA" means "Completely Automated Public Turing test to tell Computers and Humans Apart"? I expect some readers aren't aware of the direct connection w/ AI and CAPTCHA because they are unaware of the history. We should probably also link to the Wikipedia article on the topic.

If we are going to mention CAPTCHA, we should use it as one example of "things that computers weren't capable of doing before, but are DEFINITELY capable of doing now".

Reply:
I'd be very happy with linking to e.g. Wikipedia, and now I look at it more I think there's scope for more editing. I'm not sure that spelling the acronym out except in the context of a detailed explanation is worth it - the terms seems more like radar or laser than TLA to me...

```suggestion
[=verifiable credentials=]. 
Therefore, using such tasks to determine a distinction between human activity and an automated "bot",
as commonly done with a [CAPTCHA](https://en.wikipedia.org/wiki/CAPTCHA) today, 
might be no more accurate in the face of AI than current approaches.
```

Feel free to edit further. I